taskslist/app/controllers/tasks_controller.rb

```diff
@@ -31,12 +31,20 @@ def complete
 
   def edit
     @task = Task.find(params[:id])
-    @edit = true
-    render 'tasks/new'
+  end
+
+  def update
+    update_name = task_params[:task][:name]
+    update_desc = task_params[:task][:description]
+    task = Task.find(params[:id])
+    task.update(name: update_name, description: update_desc)
```

Comment:
Because we have `task_params` which uses the `permit` method on our `params` hash, we can actually pass the Task "sub-hash" directly to `update` like so: `task.update(task_params[:task])`.